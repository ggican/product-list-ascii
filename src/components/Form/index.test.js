import React from "react";
import { SelectOption } from "./index";

describe("SelectOption Component", () => {
    it("renders and create snapshot <SelectOptions />", () => {
        expect(
            toJson(
                <SelectOption
                    onGetValue={() => {}}
                    label="Select Options"
                    value="ikhsan"
                    options={[{ value: "ikhsan", label: "ikhsan" }]}
                ></SelectOption>,
            ),
        ).toMatchSnapshot();
    });
    it("renders shallow is success <SelectOption />", () => {
        expect(
            shallow(
                <SelectOption
                    onGetValue={() => {}}
                    label="Select Options"
                    value="ikhsan"
                    options={[{ value: "ikhsan", label: "ikhsan" }]}
                ></SelectOption>,
            ).exists(),
        ).toBe(true);
    });
});
