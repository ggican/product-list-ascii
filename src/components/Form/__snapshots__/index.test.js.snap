// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectOption Component renders and create snapshot <SelectOptions /> 1`] = `undefined`;
